packages/starlight/style/reset.css

```diff
@@ -40,7 +40,7 @@ h4,
 h5,
 h6,
 code {
-	overflow-wrap: anywhere;
+	background-color: rgba(255, 0, 0, 0.2);
```

Comment:
I've been through the entire doc, the only elements that are displayed in red (old rule) are the page main titles, and the "On this page" area (see https://starlight-git-fork-julien-deramond-main-jd-dc421d-astrodotbuild.vercel.app/getting-started/).

![Screenshot 2024-05-17 at 09 26 05](https://github.com/withastro/starlight/assets/17381666/04467749-f925-4332-a162-6ab3f61ce1e9)

The rest in green works well.

One solution would probably be to keep that in `markdown.css` and add specific rules for this main title and "On this page" area. What do you think @delucis? Want me to try this approach in this PR?